package.json

```diff
@@ -1,34 +1,44 @@
 {
     "name": "@capacitor-community/background-geolocation",
-    "version": "1.2.20",
+    "version": "1.2.21",
     "description": "Receive geolocation updates even while the app is in the background.",
-    "repository": {
-        "type": "git",
-        "url": "https://github.com/capacitor-community/background-geolocation"
-    },
-    "license": "MIT",
-    "author": "James Diacono",
-    "types": "definitions.d.ts",
     "files": [
         "android/build.gradle",
         "android/settings.gradle",
         "android/proguard-rules.pro",
         "android/gradle/wrapper/gradle-wrapper.properties",
         "android/gradle.properties",
-        "ios/Plugin/Plugin.*",
         "android/src/main/",
-        "ios/Podfile*",
-        "ios/Plugin/Info.plist",
+        "ios/Sources",
+        "ios/Tests",
+        "Package.swift",
         "CapacitorCommunityBackgroundGeolocation.podspec",
         "definitions.d.ts"
     ],
+    "keywords": [
+        "capacitor",
+        "plugin",
+        "native",
+        "geolocation",
+        "background"
+    ],
+    "author": "James Diacono",
+    "license": "MIT",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/capacitor-community/background-geolocation"
+    },
+    "bugs": {
+      "url": "https://github.com/capacitor-community/background-geolocation/issues"
+    },
```

Comment:
Is there any point in including a "bugs" field? If you really feel it is necessary, please correct the indentation.

Reply:
removed!